script/cibuild-node

```diff
@@ -3,4 +3,11 @@
 set -e
 
 echo "Running Node tests..."
+
+# We want to run script/build on CI to ensure things actually build, but
+# we don't want to have to run script/build locally every time we run tests
+if [ -n "$CI" ]; then
+  script/build
+fi
+
 npm test
```

Comment:
We shouldn't need `npm test` here. I believe `npm run build` runs test before building.